app/tractdbConfig.js

```diff
@@ -10,3 +10,10 @@ module_tractdbConfig.constant(
 module_tractdbConfig.config(['$httpProvider', function ($httpProvider) {
   $httpProvider.defaults.withCredentials = true;
 }]);
+
+module_tractdbConfig.config(['$locationProvider', function($locationProvider) {
```

Comment:
It basically exposes parameters passed to the url via `get`, which is how Strava passes the authentication code back to the client. `enabled` turns it on, `requireBase` removes the requirement removes the requirement of a "base" URL and everything has to be relative to it. It seems useful for a lot of the endpoints tractdb is designed to be supporting.

More detail here: [https://docs.angularjs.org/guide/$location](https://docs.angularjs.org/guide/$location)